src/dune_rules/workspace.ml

```diff
@@ -586,7 +586,7 @@ let default_step1 clflags =
   { Step1.t = lazy t; config = t.config }
 
 let load_step1 clflags p =
-  Io.with_lexbuf_from_file p ~f:(fun lb ->
```

Comment:
I thought we've hidden `Io.with_lexbuf_from_file` so that it could only be accessed as `Io.Untracked.with_lexbuf_from_file` but apparently we didn't. Could you do it while you're looking at this code?

Reply:
It's because the module that disable all this stuff isn't exported from `Dune_engine`. I'll fix that in a separate PR.